### ISSUE TITLE
save just the output result?

### ISSUE THREAD

**Issue by vurt72** [MooreThreads/Moore-AnimateAnyone#157]
Why does it add the references to the output video?? Who has even any interest in having that added, it just makes the output rather worthless. 

I am using this through Visions of Chaos, but i don't think he's fiddled with the output saving... sorry if i'm wrong.

**Comment by SoftologyPro** [comment]
Yes, please add the option to only save the output without the source image and please animations. 

I tried some hacks to save only the output but the final movie still has all 3 parts combined. 

Maybe a checkbox "Full output" to save the current results? But have it unchecked by default.
